rero_ils/modules/libraries/tasks.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Tasks on `Library` resource."""
+
+from celery import shared_task
+from invenio_cache import current_cache
+
+
+@shared_task(bind=True, name='library-calendar-changes-update-loans')
+def calendar_changes_update_loans(self, record_data):
+    """Task used to update...
+
```

Comment:
Can you explain that you considers only a day that becomes closes and not the reverse?

Reply:
I tried to be more explicit with my french_english.... :-)